src/peers/__test__/peers.test.tsx

```diff
@@ -0,0 +1,24 @@
+import Enzyme, { shallow } from 'enzyme'
+import Adapter from 'enzyme-adapter-react-16'
+import toJSON from 'enzyme-to-json'
+import React from 'react'
+import mockPeersData from './mockData'
+
+import Peers from '../peers'
+
+Enzyme.configure({ 
```

Comment:
As I commented on the other PR..
Rather than having this in every component test unit, it could be configured to run when they are loaded using the jest `setupFiles` configuration option in `jest.config.js`